manifests/job.pp

```diff
@@ -34,6 +37,7 @@
   $jobname  = $title,
   $enabled  = 1,
   $ensure   = 'present',
+  $folder   = undef,
```

Comment:
Param validation is currently inconsistent in this module (#392) but I'm requesting that all new params have validation.